### ISSUE TITLE
Forum Bug

### ISSUE THREAD

**Issue by herbertmilhomme** [herbertmilhomme/HerbertMilhomme#4]
Forum gives an alert error, and redirects after closing prompt.

Error happens on forum category/thread page, and redirects into topic list.

![bug4](https://cloud.githubusercontent.com/assets/15901670/18032682/fd678e28-6cda-11e6-867d-a56d950e7fe2.PNG)

**Comment by herbertmilhomme** [comment]
~~The problem is the site has two different forum layouts. But the javascript/css is overlapping on the two styles.~~

No wait, that wasn't the problem. The problem was that i attempted to have a table to organize the list of topics and thread, so that a user can sort, and filter the selection better. But the table javascript wasnt working with the layout i had set up.

**Comment by herbertmilhomme** [comment]
I fixed part of the bug that prevented the layout from not loading the jquery layout design. I modified the html `id`, and other tags. Then modified the javascript parameters to match up some of the information requested. But the same error occurs. This time, the good news is, when the "ok" button is hit, the page refreshes with the correct presentation.

I'm confused on why it's referring to a "row 0." **THAT'S** truly the issue here. i may have to ignore it, by modifying the core javascript to not prompt the error, if i can't figure this one out.
